### DISCUSSION TITLE
Trouble Installing

### DISCUSSION THREAD

**Discussion by ethapet** [draivin/hsnips#151] [Q&A] [answered]
Hey.

This is probably an easy fix, but I can't get the extension to work. I have created a file in the "C:\Users/peter\AppData\Roaming\Code\User\hsnips" folder called latex.hsnips and downloaded a bunch of snippets from this repository https://github.com/gruvw/latex-setup.

But when I go to type in my latex document nothing happens. Is there a final step to start hypersnips that I'm missing? Do I need to reload something?

Also the settings document looks like: 
{
    "workbench.colorTheme": "Coder Coder Dark Theme",
    "liveServer.settings.donotShowInfoMsg": true,
    "files.autoSave": "afterDelay",
    "hsnips.hsnipsPath": ""C:\Users\peter\AppData\Roaming\Code\User\hsnips""
}

Which is the same path I have the file in. Really want to start using this extension. Looks very useful.

**Comment by ethapet** [comment, marked as answer]
Answering my own question here. 

Didn't exactly know where to run those commands (so new to VS Code) so kept on running them in power shell.
Held control+shift+p and ran it in that window. Directory was defaulting to my documents folder. Ended up just clearing the path to send it back to the on in %appdata%/roaming/user/Code/hsnips

{
"workbench.colorTheme": "Coder Coder Dark Theme",
"liveServer.settings.donotShowInfoMsg": true,
"files.autoSave": "afterDelay",
"hsnips.hsnipsPath": ""
}

And ran the reload snips command. All is well now.

**Reply by perseusproject** [reply]
I am currently facing the same issue. With 

`"hsnips.hsnipsPath": "" `

or with `hsnips.hsnipsPath": "C:/Users/HP/AppData/Roaming/Code/User/hsnips"`,

 the extension does not seem to work. I even copied the all.hsnips file in the two folders so that it should work but it does not. Have you done anything else to make it work ?

**Reply by ethapet** [reply]
hey. i missed this message. you need to run the hsnips command in the vscode command pallete i think. hit ctrl+p in vscode and run the HyperSnips: Open snippets directory command. I can't really remember how exactly I did it, but changing where the folder was made no difference until it was run in vscode to link it. I have to set this up again for Python snippets (have been using it exclusively for LaTeX this past semester).
